app/templates/components/doi-alternate-identifier.hbs

```diff
@@ -0,0 +1,21 @@
+{{#if (gt index 0)}}
+  <div class="input-group">
+    <input type="text" class="form-control alternate-identifier-field" value={{fragment.identifier}} oninput={{action "updateIdentifier" value="target.value"}} data-test-alternate-identifier/>
+    <span class="input-group-btn">
+      <BsButton @outline={{true}} @onClick={{action "deleteIdentifier" index}}>{{fa-icon "trash"}}</BsButton>
+    </span>
+  </div>
+  <div class="help-block help-block-fragment">An identifier or identifiers other than the primary Identifier applied to the resource being registered.</div>
+{{else}}
+  <input type="text" class="form-control alternate-identifier-field" value={{fragment.identifier}} oninput={{action "updateIdentifier" value="target.value"}} data-test-alternate-identifier/>
+  <div class="help-block help-block-fragment">An identifier or identifiers other than the primary Identifier applied to the resource being registered.</div>
+{{/if}}
+
+  <div class="input-label">
```

Comment:
While I know we dont use the bootstrap ember validations direct components everywhere, this code could just use the regular form.element control from that library rather than using classes. Not a hard requirement just a thought. I also mention it in case im missing something with this setup which is why the validations are not appearing.

Reply:
That's an interesting point let me check that

Reply:
Although the documentation does point towards this direction. makes no difference and it gives me same errors about `func` not being a function.

Nevertheless,  I think i found it. So validations do not work at all for fragments. But the errors are caught with this helper